datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py

```diff
@@ -33,13 +33,16 @@ class PDHBaseCheck(AgentCheck):
 
     def __init__(self, *args, **kwargs):  # To support optional agentConfig
         # TODO: Change signature to (self, name, init_config, instances, counter_list) once subclasses have been edited
+        counter_list = kwargs.pop('counter_list', args[-1])  # type: List[List[str]]
+        if 'counter_list' not in kwargs:
```

Comment:
We'll need to change the logic here, this condition is always True now.

Reply:
I'll leave it as it is, it does not hurt when in kwargs